test/local/token_expiry_tests.js

```diff
@@ -6,7 +6,7 @@ var test = require('../ptaptest')
 var TestServer = require('../test_server')
 var Client = require('../client')
 
-process.env.PASSWORD_CHANGE_TOKEN_TTL = '-1'
+process.env.PASSWORD_CHANGE_TOKEN_TTL = '1'
```

Comment:
@rfk @vbudhram i suspect some change in tap or convict has a collision with setting `env` variables and then starting a server with a particular config. it seems the config is either set in some other test and is not 'reset' here or some other ugly behaviour. I personally would rather have a way to set the config value using something else rather than `process.env`.